00:00:00.016894 main     VirtualBox COM Server 6.1.12 r139181 win.amd64 (Jul 10 2020 22:02:01) release log
00:00:00.016899 main     Log opened 2020-09-26T16:11:40.189184600Z
00:00:00.016901 main     Build Type: release
00:00:00.016911 main     OS Product: Windows 10
00:00:00.016913 main     OS Release: 10.0.18363
00:00:00.016914 main     OS Service Pack: 
00:00:00.151103 main     DMI Product Name: X456URK
00:00:00.166952 main     DMI Product Version: 1.0       
00:00:00.167415 main     Firmware type: BIOS
00:00:00.167429 main     Host RAM: 8063MB (7.8GB) total, 2479MB (2.4GB) available
00:00:00.167993 main     Executable: C:\Program Files\Oracle\VirtualBox\VBoxSVC.exe
00:00:00.167994 main     Process ID: 2720
00:00:00.167995 main     Package type: WINDOWS_64BITS_GENERIC
00:00:00.329914          VirtualBox: object creation starts
00:00:00.332885          Home directory: 'C:\Users\R154NG\.VirtualBox'
00:00:00.333675          Installed Drivers:
00:00:00.343043            C:\Windows\system32\DRIVERS\VBoxNetLwf.sys (Version: 6.1.12.39181)
00:00:00.346653            C:\Windows\system32\DRIVERS\VBoxUSBMon.sys (Version: 6.1.12.39181)
00:00:00.349969            C:\Windows\system32\DRIVERS\VBoxDrv.sys (Version: 6.1.12.39181)
00:00:00.354758            C:\Windows\system32\DRIVERS\VBoxNetAdp6.sys (Version: 6.1.12.39181)
00:00:00.372695          Loading settings file "C:\Users\R154NG\.VirtualBox\VirtualBox.xml" with version "1.14-windows"
00:00:00.388474          usbLibGetDevices: Starting USB device enumeration
00:00:00.392055          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&152da58&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 6
00:00:00.392228          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&152da58&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 8
00:00:00.394631          usbLibGetDevices: Found 3 USB devices, 0 captured
00:00:00.396196          USB: Unknown USB device detected (idVendor: 0x0000, idProduct: 0x0538)
00:00:00.399475          USB: Unknown USB device detected (idVendor: 0x0bda, idProduct: 0x57de)
00:00:00.399992          USB: Unknown USB device detected (idVendor: 0x13d3, idProduct: 0x3490)
00:00:00.586208          HostDnsMonitor: initializing
00:00:00.611534          HostDnsMonitor: updating information
00:00:00.612833          HostDnsMonitor: old information
00:00:00.612835            no server entries
00:00:00.612836            no domain set
00:00:00.612836            no search string entries
00:00:00.612837          HostDnsMonitor: new information
00:00:00.612837            server 1: 118.98.44.100
00:00:00.612838            server 2: 118.98.44.10
00:00:00.612839            no domain set
00:00:00.612840            no search string entries
00:00:00.711179          VD: VDInit finished with VINF_SUCCESS
00:00:02.951716          Loading settings file "F:\Server\Debian\debian\debian.vbox" with version "1.16-windows"
00:00:02.962220          VirtualBox: object created
00:00:17.262825          Saving settings file "C:\Users\R154NG\.VirtualBox\VirtualBox.xml" with version "1.14-windows"
00:00:17.285043          Saving settings file "C:\Users\R154NG\.VirtualBox\VirtualBox.xml" with version "1.14-windows"
00:00:35.327584          Launched VM: 75458976 pid: 3764 (0xeb4) frontend: GUI/Qt name: debian
00:00:41.022867          ERROR [COM]: aRC=VBOX_E_OBJECT_NOT_FOUND (0x80bb0001) aIID={70401eef-c8e9-466b-9660-45cb3e9979e4} aComponent={ExtPackManagerWrap} aText={No extension pack by the name 'Oracle VM VirtualBox Extension Pack' was found}, preserve=false aResultDetail=0
00:00:41.565536          ERROR [COM]: aRC=VBOX_E_IPRT_ERROR (0x80bb0005) aIID={85632c68-b5bb-4316-a900-5eb28d3413df} aComponent={SessionMachine} aText={Saved screenshot data is not available (VERR_NOT_SUPPORTED)}, preserve=false aResultDetail=-37
00:36:45.212583          Saving settings file "F:\Server\Debian\debian\debian.vbox" with version "1.16-windows"
00:38:19.685960          Saving settings file "C:\Users\R154NG\.VirtualBox\VirtualBox.xml" with version "1.14-windows"
00:38:19.697091          Saving settings file "C:\Users\R154NG\.VirtualBox\VirtualBox.xml" with version "1.14-windows"
00:38:25.215712 main     VirtualBox: object deletion starts
00:38:25.216767 main     ERROR [COM]: aRC=VBOX_E_OBJECT_IN_USE (0x80bb000c) aIID={ad47ad09-787b-44ab-b343-a082a3f2dfb1} aComponent={MediumWrap} aText={Medium 'F:\Server\Debian\debian\debian.vdi' cannot be closed because it is still attached to 1 virtual machines}, preserve=false aResultDetail=0
00:38:25.219862 main     HostDnsMonitor: shutting down ...
00:38:25.219970 main     HostDnsMonitor: shut down
00:38:25.222786 Watcher  ERROR [COM]: aRC=E_ACCESSDENIED (0x80070005) aIID={d0a0163f-e254-4e5b-a1f2-011cf991c38d} aComponent={VirtualBoxWrap} aText={The object is not ready}, preserve=false aResultDetail=0
00:38:25.223677 main     VirtualBox: object deleted
